config/config.exs

```diff
@@ -0,0 +1,12 @@
+import Config
+
+case config_env() do
+  :test ->
```

Comment:
mox typically allows you to define a mock per test, rather than injecting the mock module in the config holistically. This feels like something that could easily burn you in the future as your tests would pass but dev could break.

There's also a cool library for creating mocks I've been playing with recently for the multi-cohort project.

https://search.brave.com/search?q=github+mock+elixir&source=web